backend/pkg/api/enums/validator_dashboard_enums.go

```diff
@@ -170,6 +170,35 @@ var VDBBlocksColumns = struct {
 	VDBBlockProposerReward,
 }
 
+type VDBBlocksSearches struct {
+	BasicSearch
+	Index     bool
+	PublicKey bool
+	Group     bool
+}
+
+func (s VDBBlocksSearches) GetSearches() []SearchType {
+	return []SearchType{
+		Integer,
+		ValidatorPublicKeyWithPrefix,
+		Name,
+	}
+}
```

Comment:
As discussed, we don't really need to provide a list of search types if we always check against every search type and just ignore them in `SetSearchType`

Reply:
It's optional now